test/manual/definitions/apm/node/node-linux2.json

```diff
@@ -0,0 +1,37 @@
+{
+  "global_tags": {
+    "owning_team": "OpenSource",
+    "Environment": "development",
+    "Department": "Product",
+    "Product": "Virtuoso"
+  },
+  "resources": [
+    {
+      "id": "nodelinux2",
+      "display_name": "NodeLinux2Host",
+      "provider": "aws",
+      "type": "ec2",
+      "size": "t3.micro"
+    }
+  ],
+  "services": [
+    {
+      "id": "nodetron",
+      "source_repository": "https://github.com/newrelic/demo-nodetron.git",
+      "deploy_script_path": "deploy/linux/roles",
+      "port": 5001,
+      "destinations": [
+        "nodelinux2"
+      ]
+    },
+    {
+      "destination_filepath": "engine/cronjob.json",
+      "content": [
+        {
+          "frequency": "* * * * *",
+          "job": "for ((i=1;i<=30;i++)); do curl '[service:app1:url]/'; curl '[service:app1:url]/api/inventory'; sleep 1; done"
```

Comment:
there is no `app1` in this config, so the field should be referenced using `nodetron` for the service_id [service:nodetron:url]

Reply:
Good catch. Fixed.